crates/bevy_scene/src/dynamic_scene.rs

```diff
@@ -76,17 +78,79 @@ impl DynamicScene {
         scene
     }
 
+    /// Create a new dynamic scene from a given world at a given root.
```

Comment:
This needs more explanation about what a root is, and how you would select an appropriate one :)